x-pack/plugins/session_view/public/components/ProcessTreeNode/index.tsx

```diff
@@ -229,7 +229,12 @@ export function ProcessTreeNode({
 
   return (
     <>
-      <div data-id={id} key={id + searchMatched} css={styles.processNode}>
+      <div
+        data-id={id}
+        key={id + searchMatched}
+        css={styles.processNode}
+        data-test-subj="processTreeNode"
```

Comment:
nitpick: I think `data-test-subj`s use kebab case

Reply:
it's camelCase: https://github.com/elastic/kibana/blob/main/STYLEGUIDE.mdx#camel-case-id-and-data-test-subj